heliopy/data/util.py

```diff
@@ -859,7 +854,8 @@ def _get_cdf_vars(cdf):
 
 def _fillval_nan(data, fillval):
     try:
-        data[data == fillval] = np.nan
+#        data[data == fillval] = np.nan
+        data[np.isclose(data, fillval + np.zeros_like(data))] = np.nan
```

Comment:
Suggestion for avoiding rounding errors with very large fillvals (-1e31), which resulted in fillval not tracked.

This doesn't fix erroneous fillvalues though: magplasma_l2 announces -1e31 as fillvalue but it seems that they are using -1e30 instead... Could this be resolved by resurrecting badvalues (or should we inform them and hope they fix it timely?)